package.json

```diff
@@ -151,6 +151,7 @@
     "jest-haste-map": "^24.8.1",
     "jest-resolve": "^24.8.0",
     "jest-serializer-vue": "^2.0.2",
+    "luxon": "^1.17.2",
```

Comment:
What for is luxon needed?

Reply:
The datepicker uses luxon (instead of moment) for date-"stuff". For some reason, it's not part of the dependencies, so it has to be added manually.

See https://www.npmjs.com/package/vue-datetime#bundler-webpack-rollup